docs/content/manual/pre-release/cluster-restore/backup-block-deletion.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Recover Longhorn volumes after Rancher cluster restore
+---
+
+Notice that the behaviors will be different if the cluster node roles are different. e.g., A cluster contains 1 dedicated master node + 3 worker node is different from a cluster contains 3 nodes which are both master and worker.
+This test may need to be validated for both kind of cluster.
+
+1. Launch a Longhorn system. Deploy one more compatible engine image.
+2. Prepare some volumes with workloads:
+    1. Create and attach volume A via UI. Write some data and do some snapshot operations. (Validate 1 case: The volume deleted after the cluster snapshot will be restored but is invalid)
+    2. Deploy a single pod with volume B. Write some data, do some snapshot operations, and create some backups. (Validate 3 cases: <1> data modification won't crash the whole volume; <2> backup info will be resynced; <3> users need to manual restart the single pod)
+    3. Deploy a StatefulSet with volume C. Write some data and do some snapshot operations. (Validate 1 case: Users need to manually recover the volume if all existing replicas are replaced by new replicas)
+    4. Deploy a StatefulSet with volume D. Write some data and do some snapshot operations. (Validate 2 cases: <1> volume can be recovered automatically if some replicas are removed and some new replicas are replenished; <2> snapshot info will be resynced;)
+    5. Deploy a Deployment with volume E. Write some data and do some snapshot operations. (Validate 4 cases: <1> engine upgrade; <2> offline expansion)
+3. Create a cluster snapshot via Longhorn.
+4. Do the followings before the restore:
+    1. Delete volume A.
+    2. Write more data to volume B and create more backups.
+    3. Remove all current replicas one by one for volume C. Then all replicas of volume C are new replicas.
+    4. Remove some replicas for volume D. Do snapshot creation, deletion, and revert.
+    5. Sacle down the workload. Upgrade volume E from the default image to another engine image. And do expansion.
+    6. Create and attach volume F via UI. Write some data and do some snapshot operations. (Validate 1 case: Users need to manuall recover the volume if it's created after the cluster snapshot)
+5. Restore the cluster. 
+6. Check the followings according to the doc:
+    1. Volume A is back. But there is no data in it. And users can re-delete it.
```

Comment:
I do not understand why no data in volume A after the cluster snapshot restore.

Reply:
Volume A is actually removed in step4-1 and all the data is cleaned up. The cluster restore just recover the CR objects of this volume only.